test_app/package.json

```diff
@@ -18,6 +18,6 @@
     "echoMe": "bin/echoMe"
   },
   "dependencies": {
-    "binwrap": "^1.0.0"
+    "binwrap": ".."
```

Comment:
This shouldn't need to change to a path, but probably should change to the next version number.  The way the test projects are set up here, `npm` is able to resolve it.